API-reference.md

```diff
@@ -332,6 +333,11 @@ interface IConfig {
     // If true, the SDK will track all [Browser Link](https://docs.microsoft.com/en-us/aspnet/core/client-side/using-browserlink) requests. 
     // Default: false
     isBrowserLinkTrackingEnabled: boolean;
+
+    // AppId is used for the correlation between AJAX dependencies happening on the client-side with the server-side requests. 
+    // When Beacon API is enabled it can not be used automatically, but can be set manually in the configuration.
+    // Default: null
+    appId: string;
```

Comment:
same comment as before. It needs to be made a hash map so mapping between key-appid can be registered, not a single ikey

Reply:
I would disagree here - you can set only one configuration instance and only one ikey in this object, so logically it's correct to have one appId related to that ikey. As a user I would be really confused by having a map here.
I've opened issue and it'll require additional design discussions: Improve multi-ikey support #576

Reply:
we can have a single `appId` here. However there should be mechanism to provide mapping for more keys somehow. For the multi-key users.